test/integration/handlers/handlers.test.js

```diff
@@ -48,29 +51,43 @@ const TransferEventType = Enum.transferEventType
 const TransferEventAction = Enum.transferEventAction
 
 const debug = false
-const delay = 8000 // milliseconds
+const rebalanceDelay = 10000
+const retryDelay = 500
+const retryCount = 40
+const retryOpts = {
+  retries: retryCount,
+  minTimeout: retryDelay,
+  maxTimeout: retryDelay
+}
+
 let testData = {
   amount: {
     currency: 'USD',
     amount: 110
   },
   payer: {
-    name: 'payer',
+    name: 'payerfsp',
     limit: 500
   },
   payee: {
-    name: 'payee',
+    name: 'payeefsp',
     limit: 300
   },
+  endpoint: {
+    base: 'http://localhost:1080',
+    email: 'test@example.com'
+  },
   now: new Date(),
   expiration: new Date((new Date()).getTime() + (24 * 60 * 60 * 1000)) // tomorrow
 }
 
 const prepareTestData = async (dataObj) => {
   let payer = await ParticipantHelper.prepareData(dataObj.payer.name, dataObj.amount.currency)
   let payee = await ParticipantHelper.prepareData(dataObj.payee.name, dataObj.amount.currency)
-  const kafkacat = `kafkacat bash command to trace kafka topics:\nPAYER=${payer.participant.name}; PAYEE=${payee.participant.name}; GROUP=grp; T=topic; kafkacat -b localhost -G $GROUP $T-$PAYER-transfer-prepare $T-$PAYER-position-prepare $T-$PAYER-position-fulfil $T-transfer-fulfil $T-$PAYEE-position-fulfil $T-transfer-transfer $T-notification-event $T-$PAYEE-transfer-prepare $T-$PAYEE-position-abort $T-$PAYER-position-abort $T-$PAYEE-position-prepare`
-  if (debug) console.log(kafkacat)
+
+  const kafkacat = `GROUP=abc; T=topic; TR=transfer; kafkacat -b localhost -G $GROUP $T-$TR-prepare $T-$TR-position $T-$TR-fulfil $T-$TR-get $T-admin-$TR $T-notification-event`
```

Comment:
You do know you can import kafkacat as a library instead of calling a bash command right?

Reply:
This is just output for debugging purposes. Should I remove it?